content/concepts/transactions/payments/checks.md

```diff
@@ -7,94 +7,45 @@ labels:
 
-* Checks do not lock up or set aside any funds. The XRP involved in payment channels and escrow cannot be spent until it is redeemed with a claim provided by the sender (payment channels), or released by an expiration or crypto-condition (escrow).
+3. The recipient submits a `CheckCash` transaction that transfers the funds and destroys the `Check` object.
+    **Note:** Recipients have two options for cashing checks.
+        - Exact Amount: They specify an exact amount to cash that doesn't exceed the check maximum.
+        - Flexible Amount: They specify a minimum amount to cash and the XRP Ledger delivers as much as possible up to the check maximum. If the sender doesn't have the funds to meet the minimum, the transaction fails.
 
-* You can send XRP to yourself through escrow. You cannot send checks to yourself.
+4. If the check expires before the receiver cashes the check, the `Check` object remains until someone cancels it.
 
 
-**Note:** The Checks amendment changes the expiration behavior of the `OfferCreate` transaction. 
+## Use Cases
 
-<!-- For more information, see [Offer Expiration](offers.html#offer-expiration). -->
+### Deposit Authorization
 
-
-## Why Checks?
-
-Traditional paper checks allow people to transfer funds without immediately exchanging physical currency. XRP Ledger Checks allow people to exchange funds asynchronously using a process that is familiar to and accepted by the banking industry.
-
-XRP Ledger Checks also solve a problem that is unique to the XRP Ledger: they allow users to reject unwanted payments or accept only part of a payment. This is useful for institutions that need to be careful about accepting payments for compliance reasons.
-
-
-### Use Case: Payment Authorization
-
-**Problem:** To comply with regulations like BSA, KYC, AML, and CFT, financial institutions must provide documentation about the source of funds they receive. Such regulations seek to prevent the illicit transfer of funds by requiring institutions to know the source and destination of all payments processed by the institution. Because of the nature of the XRP Ledger, anyone could potentially send XRP (and, under the right circumstances, tokens) to an institution's account on the XRP Ledger. Dealing with such unwanted payments adds significant cost and time delays to these institutions' compliance departments, including potential fines or penalties. <!-- SPELLING_IGNORE: cft -->
+**Problem:** To comply with regulations like BSA, KYC, AML, and CFT, financial institutions must provide documentation about the sources of funds they receive. These regulations seek to prevent illicit activity by requiring institutions to track the source and destination of all payments they process. Because of the nature of the XRP Ledger, anyone can send XRP to an institution's account on the ledger; to prevent this behavior, they'll enable deposit authorization to only receive funds they explicitly approve. <!-- SPELLING_IGNORE: cft -->
```

Comment:
If we're not going to link out to the definitions of "BSA, KYC, AML, and CFT," I don't see the purpose in talking about them here. I might just say "To comply with banking regulations, financial institutions must provide...." The TLAs are just a distraction, to my. mind.